src/domain/occurrence/__tests__/mocks.ts

```diff
@@ -0,0 +1,769 @@
+export const eventResult = {
```

Comment:
Maybe you could explicitly add types to these mock responses? It might help if the data structure of the graphql query responses changes at some point

Reply:
Good point